Python/Product/Common/Strings.resx

```diff
@@ -2061,7 +2061,7 @@ Expand selection to the full expression?</value>
     <comment>Do not translate 'PyPI'</comment>
   </data>
   <data name="CondaExtensionCondaInstallFrom" xml:space="preserve">
-    <value>Run command: conda install {0}</value>
+    <value>Run command: Conda install {0}</value>
```

Comment:
I wouldn't capitalize conda here, it is meant as the name of the command line executable file, which is all lowercase.